docs/geedocs/5.2.5/answer/7160006.html

```diff
@@ -87,6 +87,13 @@ <h5>
               <code>gestartdaemon</code> and <code>gestopdaemon</code> utilities have been removed.
             </td>
           </tr>
+	  <tr>
+            <td>1038</td>
+            <td><code>genewmapdatabase</code> does not allow flat imagery to be used with mercator databses</td>
+            <td><code>genewmapdatabase</code> now looks for flat imagery projects as well as 
+               mercator imagery projects when adding imagery projects to mercator databases. 
+               Mercator imagery projects are given priority over flat imagery projects in the case of               a naming collision. Flat databases can still only use flat imagery projects.</td>
```

Comment:
there is white space in the middle of the phrase.

Reply:
Cleaned it up.